.github/workflows/run_system_tests.yml

```diff
@@ -22,7 +22,16 @@ jobs:
     steps:
       - name: Check out repo
         uses: actions/checkout@v4
+      - name: Cache virtualenvs
```

Comment:
Caching isn't working on measlinkbot:
```
Post job cleanup.
"C:\Program Files\Git\usr\bin\tar.exe" --posix -cf cache.tgz --exclude cache.tgz -P -C C:/actions-runner/[1](https://github.com/ni/measurementlink-python/actions/runs/6675067827/job/18142674125#step:11:1)/_work/measurementlink-python/measurementlink-python --files-from manifest.txt --force-local -z
/bin/sh: line 1: gzip: command not found
/usr/bin/tar: cache.tgz: Cannot write: Broken pipe
/usr/bin/tar: Child returned status 1[2](https://github.com/ni/measurementlink-python/actions/runs/6675067827/job/18142674125#step:11:2)7
/usr/bin/tar: Error is not recoverable: exiting now
Warning: Failed to save: "C:\Program failed with error: The process 'C:\Program Files\Git\usr\bin\tar.exe' failed with exit code 2
```

Reply:
Fixed by changing measlinkbot to include the UNIX tools when installing Git and also install zstandard.